website/docs/d/is_security_group_rules.html.markdown

```diff
@@ -0,0 +1,54 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_is_security_group_rules"
+description: |-
+  Get information about SecurityGroupRuleCollection
+subcategory: "Virtual Private Cloud API"
+---
+
+# ibm_is_security_group_rules
+
+Provides a read-only data source for SecurityGroupRuleCollection. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
```

Comment:
Refer the resource of security group rule and add the reference here

Reply:
done.

Reply:
done.